examples/plotting/zoom_map_example.py

```diff
@@ -0,0 +1,66 @@
+"""
+=========================
+Zoomed-in Cutout of a Map
+=========================
+
+This example demonstrates how to plot a cutout region of a `~sunpy.map.Map` with
+connector lines that indicate the region of interest in the full-disk image.
+"""
+
+import matplotlib.pyplot as plt
+from matplotlib.patches import ConnectionPatch
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE
+
+###############################################################################
+# We will start off by defining a region of interest that we want to focus on.
+# In this case, we want to focus on the center of the Sun.
+length = 600 * u.arcsec
+x0 = 550 * u.arcsec
+y0 = -700 * u.arcsec
+
+###############################################################################
+# Now we will create a Map and then crop the region of interest.
+smap = sunpy.map.Map(AIA_171_IMAGE)
+bottom_left = SkyCoord(x0, y0, frame=smap.coordinate_frame)
+top_right = SkyCoord(x0 + length, y0+length,
+                     frame=smap.coordinate_frame)
+
+submap = smap.submap(bottom_left, top_right=top_right)
+
+fig = plt.figure(figsize=(16, 5))
+ax1 = fig.add_subplot(1, 2, 1, projection=smap)
+
+smap.plot(axes=ax1, clip_interval=(1, 99.9)*u.percent)
+
+# Draw a box on the image highlighting the region of interest.
+smap.draw_rectangle(bottom_left, color='w', height=length, width=length)
+
+# Create a second axis on the plot which will be used to create the zoomed-in image.
+ax2 = fig.add_subplot(1, 2, 2, projection=submap)
+
+submap.plot(axes=ax2, clip_interval=(1, 99.9)*u.percent)
+submap.draw_grid(grid_spacing=20*u.deg)
+
+ax2.set_title(None)
+ax2.set_ylabel('Solar Latitude')
+ax2.set_xlabel('', color='w')
+
```

Comment:
I think someone need to approve these changes as i put white color to hide some unexpected default values that i got after removing the labels. I am not sure this can be done or not. I tried many ways but nothing worked for me except this!

![image](https://user-images.githubusercontent.com/72349558/110898769-daa1d680-8325-11eb-92ff-13d66c6fdc3f.png)

Reply:
Should be ok.